src/main/java/org/folio/inventory/MarcHridSetConsumerVerticle.java

```diff
@@ -49,29 +54,26 @@ public void start(Promise<Void> startPromise) {
       .build();
     LOGGER.info("kafkaConfig: {}", kafkaConfig);
 
-    SubscriptionDefinition subscriptionDefinition = KafkaTopicNameHelper.createSubscriptionDefinition(kafkaConfig.getEnvId(),
-        KafkaTopicNameHelper.getDefaultNameSpace(), DI_SRS_MARC_BIB_INSTANCE_HRID_SET.value());
-
-    consumerWrapper = KafkaConsumerWrapper.<String, String>builder()
-      .context(context)
-      .vertx(vertx)
-      .kafkaConfig(kafkaConfig)
-      .loadLimit(loadLimit)
-      .globalLoadSensor(GLOBAL_LOAD_SENSOR)
-      .subscriptionDefinition(subscriptionDefinition)
-      .build();
+    marcBibConsumerWrapper = createConsumerByEvent(kafkaConfig, DI_SRS_MARC_BIB_INSTANCE_HRID_SET);
+    marcHoldingsConsumerWrapper = createConsumerByEvent(kafkaConfig, DI_SRS_MARC_HOLDINGS_HOLDING_HRID_SET);
 
     HttpClient client = vertx.createHttpClient();
     Storage storage = Storage.basedUpon(vertx, config, client);
     InstanceUpdateDelegate instanceUpdateDelegate = new InstanceUpdateDelegate(storage);
+    HoldingUpdateDelegate holdingUpdateDelegate = new HoldingUpdateDelegate(storage);
 
     KafkaInternalCache kafkaInternalCache = KafkaInternalCache.builder()
       .kafkaConfig(kafkaConfig).build();
     kafkaInternalCache.initKafkaCache();
 
     MarcBibInstanceHridSetKafkaHandler marcBibInstanceHridSetKafkaHandler = new MarcBibInstanceHridSetKafkaHandler(instanceUpdateDelegate, kafkaInternalCache);
+    MarcHoldingsHridSetKafkaHandler marcHoldingsHridSetKafkaHandler = new MarcHoldingsHridSetKafkaHandler(holdingUpdateDelegate, kafkaInternalCache);
+
+    marcBibConsumerWrapper.start(marcBibInstanceHridSetKafkaHandler, ConsumerWrapperUtil.constructModuleName())
+      .onSuccess(v -> startPromise.complete())
+      .onFailure(startPromise::fail);
 
-    consumerWrapper.start(marcBibInstanceHridSetKafkaHandler, ConsumerWrapperUtil.constructModuleName())
+    marcHoldingsConsumerWrapper.start(marcHoldingsHridSetKafkaHandler, ConsumerWrapperUtil.constructModuleName())
       .onSuccess(v -> startPromise.complete())
```

Comment:
You can't complete promise 2 times: after marcBibConsumerWrapper started and after marcHoldingsConsumerWrapper started. I suggest using compose after marcBibConsumerWrapper to start marcHoldingsConsumerWrapper or using ComposeFuture.